gateway/crates/gateway-binary/tests/mocks/uplink.rs

```diff
@@ -0,0 +1,33 @@
+use std::ops::Deref;
+use std::str::FromStr;
+
+use ulid::Ulid;
+
+use federated_server::UplinkResponse;
+
+#[derive(serde::Serialize, serde::Deserialize)]
+pub(crate) struct UplinkResponseMock(UplinkResponse);
```

Comment:
Not sure why you did a `UplinkResponseMock`, a single function `fn mock_uplink_response(sdl: String) -> UplinkResponse` seems to do the job in a simpler way.